src/components/link.js

```diff
@@ -49,6 +49,11 @@ export default {
         // don't redirect on right click
         /* istanbul ignore if */
         if (e.button !== 0) return
+        // don't redirect if `target="_blank"`
+        /* istanbul ignore if */
+        const target = this.$el && this.$el.attributes.getNamedItem('target')
+        if (target && target.value === '_blank') return
```

Comment:
Two things:

* Why not `const target = this.$el && this.$el.getAttribute('target')`?
* Won't this fail for `target="_blank "` (with an extra space)?